pkg/scheduler/testutil.go

```diff
@@ -1,90 +0,0 @@
-/*
```

Comment:
Review note for me: Since 3c24c99b08292a44a26af1b5b7f4450813e2bbb9 this FakeConfigurator has not been used at all.
So this removal is good for me.